cmd/iop/mandiff.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package iop
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"istio.io/operator/pkg/manifest"
+	"istio.io/pkg/log"
+)
+
+type manDiffArgs struct {
+	// compareDir indicates comparison between directory.
+	compareDir bool
+}
+
+const YamlSuffix = "yaml"
+
+func manifestDiffCmd(rootArgs *rootArgs, diffArgs *manDiffArgs) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "manifestsDiff",
```

Comment:
The convention for our CLI is dashes rather than camel i.e. manifest-diff. I would suggest diff-manifest however so that it groups together with diff-yaml (we are likely going to have this command to compare configs).

Reply:
done